docs/man/docker.1.md

```diff
@@ -319,6 +322,18 @@ for data and metadata:
       --storage-opt dm.metadatadev=/dev/vdc \
       --storage-opt dm.basesize=20G
 
+# EXEC DRIVER OPTIONS
```

Comment:
`# Examples` heading with its `Client` subhead should come after this.  "As of now and only viable" are assumed in context.

-->
# EXEC DRIVER OPTIONS

Use the **--exec-opt** flags to specify options to the exec-driver. The only
driver that accepts this flag is the *native* (libcontainer) driver. As a
result, you must also specify **-s=**native for this option to have effect. The following is the only *native* option:

#### native.cgroupdriver
Specifies the management of the container's `cgroups`. You can specify `cgroupfs` or `systemd`. If you specify `systemd` and it is not available, the system uses `cgroupfs`.